File: in_Python/1385 Find the Distance Value Between Two Arrays.py
class Solution(object):
    def findTheDistanceValue(self, arr1, arr2, d):
        """
        :type arr1: List[int]
        :type arr2: List[int]
        :type d: int
        :rtype: int
        """
        res = 0
        for i in arr1:
            keep = True
            for j in arr2:
                if abs(i-j) <= d:
                    keep = False
                    break
            if keep: res+=1
        return res

File: in_Java/1385 Find the Distance Value Between Two Arrays.java
class Solution {
    public int findTheDistanceValue(int[] arr1, int[] arr2, int d) {
        int res = 0;
        boolean f;
        for(int i : arr1){
            f = true;
            for(int j : arr2){
                if(Math.abs(i - j) <= d){
                    f = false;
                    break;
                }
            }
            if(f) res++;
        }
        return res;
    }
}